### PR TITLE
Rename project_prefix

### PR DESCRIPTION
**Pull request by z-shah**
- Rename **var.project_prefix** to be **prj** instead of *cft*
- Add **random_id** suffix to  cloudbuild_project_id to avoid potentital conflicts with parallel test runs.
- Rename terraform state bucket name to **bkt-b-tfstate** 
- Update docs

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: variables.tf [hunk 1]
```diff
@@ -62,7 +62,7 @@ variable "project_labels" {
 
 variable "project_prefix" {
   description = "Name prefix to use for projects created."
-  default     = "cft"
+  default     = "prj"
```
Comment (bharathkkb) [inline variables.tf:65]: We should keep the default here as is. We have added the `prj` default in example-foundations https://github.com/terraform-google-modules/terraform-example-foundation/blob/34c9125306271994e86a1a63980b7c7f5af01291/0-bootstrap/variables.tf#L61

File: modules/cloudbuild/main.tf [hunk 2]
```diff
@@ -15,7 +15,7 @@
  */
 
 locals {
-  cloudbuild_project_id       = var.project_id != "" ? var.project_id : format("%s-%s", var.project_prefix, "cloudbuild")
+  cloudbuild_project_id       = var.project_id != "" ? var.project_id : format("%s-%s-%s", var.project_prefix, "cloudbuild", random_id.suffix.hex)
```
Comment (bharathkkb) [inline modules/cloudbuild/main.tf:18]: Project factory module already adds a random suffix within this module https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/c8db8f3ca09ebb9a99155ac10727f01ec7fd7645/modules/cloudbuild/main.tf#L42

We should construct a `project_id` in example foundations and pass into this module via `project_id` in https://github.com/terraform-google-modules/terraform-example-foundation/blob/34c9125306271994e86a1a63980b7c7f5af01291/0-bootstrap/main.tf#L115 for more randomization

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR @z-shah 
Most of these changes makes sense to be put into example-foundations directly

File: modules/cloudbuild/main.tf (hunk 2 above)
Comment (z-shah) [reply modules/cloudbuild/main.tf]: Hey @bharathkkb 
Do you mean construct it in the same format as we do here ie. `project_prefix-cloudbuild-randomhex`
example foundation also references it here https://github.com/terraform-google-modules/terraform-example-foundation/blob/34c9125306271994e86a1a63980b7c7f5af01291/0-bootstrap/main.tf#L176

**Pull request closed without merging**